### DISCUSSION TITLE
How to add sentences for general sensor.something ids?

### DISCUSSION THREAD

**Discussion by Mattstir** [OHF-Voice/intents#1211] [_General]
Hi,

I'm quite new to under the hood HAS things.. A tired out a few sentences (in german) after updating and saw they didn't work. Therefore I wanted to contribute them here.. Checking how things are organized I now came to a point where I'm not sure what to do...

I see in all packages there are `climate_HassClimateGetTemperature` for all temp related things. So they only work for `climate.something` ids, right?
What about `sensor.something`. All my temp sensors have a `sensor.something` id in my HAS. They got these by default.. How would be the process here?

Another question. There are a lot more other interesting `sensor.xxxx` ids that I have. Like power monitoring and so on.. How could I create sentences for these? Do they all have to go into the general `homeassistant_HassGetState.yaml`?

Thanks!

**Comment by TheFes** [comment]
you can create `sensor_HassGetState.yaml` files for that.
But for support of new domains, the standard approach is to first add them to English, however it looks like someone is already working on that https://github.com/home-assistant/intents/discussions/1211

**Reply by Mattstir** [reply]
Alright, thanks! I guess I will add something then :D

**Reply by spuljko** [reply]
@Mattstir I am trying to do something like that (#1204), you are welcomed to contribute.

**Reply by Mattstir** [reply]
@spuljko  FYI I opened https://github.com/home-assistant/intents/pull/1220

**Comment by tetele** [comment]
@synesthesiam should we add `device_class`-based sentences for the `sensor` domain? Are those supported in HA?

**Comment by TheFes** [comment]
Several people started working on this (also see the links above) but nobody finalized it :)
I also planned to have a look, but I should first finalize the binary sensors for NL, and can't find the time for that at the moment.
